library/spark/spark-accumulo-library/src/main/java/uk/gov/gchq/gaffer/sparkaccumulo/operation/rfilereaderrdd/RFileReaderIterator.java

```diff
@@ -212,13 +212,13 @@ private List<IteratorSetting> getIteratorSettings() {
 
     private void close() {
         for (final SortedKeyValueIterator<Key, Value> iterator : iterators) {
-            RFile.Reader reader = null;
+            RFile.Reader reader;
             try {
                 reader = (RFile.Reader) iterator;
                 LOGGER.debug("Closing RFile.Reader {}", reader);
                 reader.close();
             } catch (final IOException e) {
-                LOGGER.error("IOException closing reader {}", reader);
+                LOGGER.error("IOException closing reader {}", e.getMessage());
```

Comment:
Why have we used 'e.getMessage' here, but removed it in other places in favour of just 'e'. My preference would be for consistency unless there is a reason for this.